### ISSUE TITLE
Optifine wont open in java binary

### ISSUE THREAD

**Issue by 314kojioyama** [sp614x/optifine#2560]
I have tried the latest OptiFine versions for 1.12.2, 1.13.2, 1.14.2, 1.14.3 and when I double click and/or right click open with java binary, nothing happens. I have installed and open all the corresponding Minecraft versions before trying to install OptiFine. I have also tried the manual file alter/replacing explained on the Minecraft forum and still OptiFine doesn't show, in version number or in game settings. I also put 1.14.3 in its own directory and launched it, then I went to OptiFine_1.14.3_HD_U_F1 in my download folder and opened with java binary, but all that happens is my cursor turns into a blue loading circle for a second then nothing. I also tried from my desktop but the OptiFine installer wont open. I currently have java 1.8_211 in my x86 java directory, but Minecraft will not launch with any other version of java other than what it comes with on my computer, because of an openGL incompatibility from win10 with the integrated intel HD chipset. would it be safe or worth trying to uninstall java 1.8_211 and replacing it with the java from my Minecraft directory to try and install OptiFine? the problem again is that OptiFine will not launch the installer even if I right click, open with, java SE binary.

**Comment by BenjaminHoegh** [comment]
have same issue

**Comment by sp614x** [comment]
Duplicate of #2612, #2258, #2170, #943, #906, #763.

You can use JarFix to restore the JAR file association:
https://download.cnet.com/Jarfix/3000-18512_4-77571216.html

**Comment by 314kojioyama** [comment]
thankyou for finally replying. I tried this as well as the reinstall of java that they recommended(broke my java launcher, as Minecraft doesn't open either now). but it still wont do. I'm going to put this on my computer for being old and dumb at this point. I have had to move my Minecraft joy to bedrock....(moan of displeasure). I am a great red stoner and love to do red stone machines. maybe now that I have been forced to bedrock I will work up the courage and do some vids on bedrock red stone and how to fix java machines.
